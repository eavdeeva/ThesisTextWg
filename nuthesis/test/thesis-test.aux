\relax 
\ifx\hyper@anchor\@undefined
\global \let \oldcontentsline\contentsline
\gdef \contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global \let \oldnewlabel\newlabel
\gdef \newlabel#1#2{\newlabelxx{#1}#2}
\gdef \newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\let \contentsline\oldcontentsline
\let \newlabel\oldnewlabel}
\else
\global \let \hyper@last\relax 
\fi

\@writefile{toc}{\contentsline {chapter}{Contents}{vii}{section*.1}}
\@writefile{toc}{\contentsline {chapter}{List of Figures}{viii}{section*.2}}
\@writefile{toc}{\contentsline {chapter}{List of Tables}{ix}{section*.3}}
\citation{virgil}
\citation{virgil}
\citation{virgil}
\@writefile{lof}{\addvspace {10pt}}
\@writefile{lot}{\addvspace {10pt}}
\@writefile{toc}{\contentsline {chapter}{\chapternumberline {1}The \AE {}NIED, Book I}{1}{chapter.1}}
\newlabel{chap:aenied}{{\M@TitleReference {1}{The \AE {}NIED, Book I}}{1}{\relax }{chapter.1}{}}
\@writefile{toc}{\contentsline {section}{\numberline {1.1}Start}{1}{section.1.1}}
\@writefile{brf}{\backcite{virgil}{{1}{1.1}{section.1.1}}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.1.1}Middle}{1}{subsection.1.1.1}}
\@writefile{brf}{\backcite{virgil}{{1}{1.1.1}{subsection.1.1.1}}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {1.1.1.1}End}{2}{subsubsection.1.1.1.1}}
\@writefile{brf}{\backcite{virgil}{{2}{1.1.1.1}{subsubsection.1.1.1.1}}}
\@writefile{lof}{\addvspace {10pt}}
\@writefile{lot}{\addvspace {10pt}}
\@writefile{toc}{\contentsline {chapter}{\chapternumberline {2}Some Tables and Figures}{3}{chapter.2}}
\@writefile{lot}{\contentsline {table}{\numberline {2.1}{\ignorespaces Arma virumque cano, Troiae qui primus ab oris Italiam, fato profugus, Laviniaque venit litora, multum ille et terris iactatus et alto vi superum saevae memorem Iunonis ob iram}}{3}{table.2.1}}
\newlabel{tab:tabular}{{\M@TitleReference {2.1}{Arma virumque cano, Troiae qui primus ab oris Italiam, fato profugus, Laviniaque venit litora, multum ille et terris iactatus et alto vi superum saevae memorem Iunonis ob iram}}{3}{Arma virumque cano, Troiae qui primus ab oris Italiam, fato profugus, Laviniaque venit litora, multum ille et terris iactatus et alto vi superum saevae memorem Iunonis ob iram\relax }{table.2.1}{}}
\@writefile{lot}{\contentsline {table}{\numberline {2.2}{\ignorespaces Arma virumque cano, Troiae qui primus ab oris Italiam, fato profugus, Laviniaque venit litora, multum ille et terris iactatus et alto vi superum saevae memorem Iunonis ob iram}}{3}{table.2.2}}
\newlabel{tab:list}{{\M@TitleReference {2.2}{Arma virumque cano, Troiae qui primus ab oris Italiam, fato profugus, Laviniaque venit litora, multum ille et terris iactatus et alto vi superum saevae memorem Iunonis ob iram}}{3}{Arma virumque cano, Troiae qui primus ab oris Italiam, fato profugus, Laviniaque venit litora, multum ille et terris iactatus et alto vi superum saevae memorem Iunonis ob iram\relax }{table.2.2}{}}
\@writefile{lof}{\contentsline {figure}{\numberline {2.1}{\ignorespaces Arma virumque cano, Troiae qui primus ab oris Italiam, fato profugus, Laviniaque venit litora, multum ille et terris iactatus et alto vi superum saevae memorem Iunonis ob iram}}{4}{figure.2.1}}
\newlabel{fig:test}{{\M@TitleReference {2.1}{Arma virumque cano, Troiae qui primus ab oris Italiam, fato profugus, Laviniaque venit litora, multum ille et terris iactatus et alto vi superum saevae memorem Iunonis ob iram}}{4}{Arma virumque cano, Troiae qui primus ab oris Italiam, fato profugus, Laviniaque venit litora, multum ille et terris iactatus et alto vi superum saevae memorem Iunonis ob iram\relax }{figure.2.1}{}}
\@writefile{lof}{\addvspace {10pt}}
\@writefile{lot}{\addvspace {10pt}}
\@writefile{toc}{\contentsline {chapter}{\chapternumberline {3}Some Math}{5}{chapter.3}}
\newlabel{chap:math}{{\M@TitleReference {3}{Some Math}}{5}{\relax }{chapter.3}{}}
\newlabel{eq:fun}{{3.3}{5}{\relax }{equation.3.3}{}}
\bibstyle{plain}
\bibdata{nuthesis}
\@writefile{lof}{\addvspace {10pt}}
\@writefile{lot}{\addvspace {10pt}}
\@writefile{toc}{\contentsline {appendix}{\chapternumberline {A}Testing, 1, 2, 3, \ldots  }{6}{appendix.A}}
\bibcite{url}{1}
\bibcite{beebebib}{2}
\bibcite{color}{3}
\bibcite{graphicx}{4}
\bibcite{keyval}{5}
\bibcite{showkeys}{6}
\bibcite{ifthen}{7}
\bibcite{uiucthesis}{8}
\bibcite{booktabs}{9}
\@writefile{toc}{\contentsline {chapter}{Bibliography}{7}{section*.5}}
\bibcite{ucthesis}{10}
\bibcite{fancyvrb}{11}
\bibcite{unhappy}{12}
\bibcite{ukfaq}{13}
\bibcite{freemathfonts}{14}
\bibcite{scrtime}{15}
\bibcite{doc}{16}
\bibcite{docstrip}{17}
\bibcite{varioref}{18}
\bibcite{Mittelbach:2004:LC}{19}
\bibcite{listings}{20}
\bibcite{nomencl}{21}
\bibcite{shortintro}{22}
\bibcite{unl-thesis-electronic}{23}
\bibcite{unl-thesis-guidebook}{24}
\bibcite{comprehensive}{25}
\bibcite{packagetutorial}{26}
\bibcite{packagewriters}{27}
\bibcite{mathpazo}{28}
\bibcite{hyperref}{29}
\bibcite{epslatex}{30}
\bibcite{productivity}{31}
\bibcite{pxfonts}{32}
\bibcite{paralist}{33}
\bibcite{microtype}{34}
\bibcite{amsfonts}{35}
\bibcite{amsmath}{36}
\bibcite{setspace}{37}
\bibcite{geometry}{38}
\bibcite{virgil}{39}
\bibcite{voss-mathmode}{40}
\bibcite{memoir}{41}
\bibcite{memoirArticle}{42}
\citation{*}
\setcounter{lastsheet}{20}
\setcounter{lastpage}{11}
