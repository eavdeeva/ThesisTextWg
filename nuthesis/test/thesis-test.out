\BOOKMARK [0][-]{section*.1}{Contents}{}
\BOOKMARK [0][-]{section*.2}{List of Figures}{}
\BOOKMARK [0][-]{section*.3}{List of Tables}{}
\BOOKMARK [0][-]{chapter.1}{The \306NIED, Book I}{}
\BOOKMARK [1][-]{section.1.1}{Start}{chapter.1}
\BOOKMARK [2][-]{subsection.1.1.1}{Middle}{section.1.1}
\BOOKMARK [3][-]{subsubsection.1.1.1.1}{End}{subsection.1.1.1}
\BOOKMARK [0][-]{chapter.2}{Some Tables and Figures}{}
\BOOKMARK [0][-]{chapter.3}{Some Math}{}
\BOOKMARK [0][-]{appendix.A}{Testing, 1, 2, 3, \203}{}
\BOOKMARK [0][-]{section*.5}{Bibliography}{}
